### ISSUE TITLE
go.mod refers to a non-existent project

### ISSUE THREAD

**Issue by yurivict** [cheat/cheat#500]
github.com/docopt/docopt-go has been renamed.

https://github.com/cheat/cheat/blob/master/go.mod#L8

**Comment by chrisallenlane** [comment]
Hi, @yurivict. Thanks for reporting this.

This is bizarre. I'm not sure what's happening here.

At first glance, it appears that you're correct - `docopt-go` has been renamed to `docopt.go`. That appears to be straightforward.

However - when I make the corresponding change in `go.mod`, and then re-build the executable, `go` changes the module's name back to `go.mod`.

It does this even when I `rm -rf vendor go.sum`. I've also tried `go clean`. No matter what I seem to do, that module gets renamed back to `docopt-go`!

So I'm confused. I really don't understand what's going on here.

As a side-effect of this ticket: I just released `cheat` version `3.0.7`. It updates dependencies. I've also configured Github's "Dependabot" to now scan this repository daily to help automatically keep our dependencies up-to-date. Likewise, I modified our build scripts to ensure that the `vendor` directory stays in-sync with the contents of `go.mod`.

But regarding `docopt-go` itself? I'm at a loss for what's happening here. I'm going to pin this issue and leave it open for a little while, to see if someone cleverer than me can solve it.
